src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -2265,6 +2265,14 @@ void updateZoneSizingEndZoneSizingCalc7(EnergyPlusData &state,
             zoneSizing.DesCoolVolFlowNoOA = zoneSizing.DesCoolVolFlow;
             zoneSizing.DesCoolMassFlowNoOA = zoneSizing.DesCoolMassFlow;
         }
+    } else {
+        for (int DDNum = 1; DDNum <= state.dataEnvrn->TotDesDays + state.dataEnvrn->TotRunDesPersDays; ++DDNum) {
+            // initialize HeatFlowSeqNoOA before any adjustments to HeatFlowSeq
+            auto &zoneSizing = zsSizing(DDNum, zoneOrSpaceNum);
+            zoneSizing.CoolFlowSeqNoOA = zoneSizing.CoolFlowSeq;
+            zoneSizing.DesCoolVolFlowNoOA = zoneSizing.DesCoolVolFlow;
+            zoneSizing.DesCoolMassFlowNoOA = zoneSizing.DesCoolMassFlow;
+        }
```

Comment:
The Cool/HeatFlowSeqNoOA variables were not initialized except for inside the IF above this else. This code initializes these variables if the conditional fails for that IF.